mdm/checkin/endpoint.go

```diff
@@ -26,8 +26,10 @@ func MakeCheckinEndpoint(svc Service) endpoint.Endpoint {
 			err = svc.TokenUpdate(ctx, req.CheckinCommand)
 		case "CheckOut":
 			err = svc.CheckOut(ctx, req.CheckinCommand)
+		case "UserAuthenticate":
```

Comment:
As implemented this change would prevent the admin user account as installed by the blueprint. No?

Reply:
I don't think so, but we should test to be sure. The user MDM checkin (UserAuthenticate) happens post user-login I think. In any case responding with a 401 can't be any better. :)